src/openrct2/Input.cpp

```diff
@@ -18,7 +18,12 @@ uint8_t gInputPlaceObjectModifier;
 widget_ref gHoverWidget;
 widget_ref gPressedWidget;
 
-uint16_t _tooltipNotShownTicks;
+int _tooltipOnTimeCounter;
+int _tooltipDisplayTimeCounter;
+int _tooltipDisplayDefaultWaitTime = 2500;
+int _tooltipDisplayShortWaitTime = 100;
```

Comment:
> I do agree that this mostly improves how tooltips work, but the code introduces many globals that should be moved to some shared class. It doesn't make sense that tooltip code is part of Input.h/cpp. These should be moved to the (new) Tooltip.h/cpp file, or even better made part of a TooltipWindow class.
> 
> The initial time for tooltips to appear seem to be longer than on current develop. The rest - showing tooltips immediately when hovering over another widget with one - feels much better.


I also prefer responsive tooltips
And there is no variable for timing, code shorter and more concise. 
However, for users who are familiar with the interface, it may be a bit of a spam-like visual obstruction.
I think it's better to apply the vanilla rct2 rule for these trivial things.
I think it's good to add options like 'off-normal-fast' for tooltip behavior in future options.